JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/AccessBean.java

```diff
@@ -0,0 +1,56 @@
+package ru.javafiddle.core.ejb;
```

Comment:
This class is absolutely equal to the one already in Development branch. And I see a lot of such files here.
Could you please filter your Pull request from such pseudo-changes (use rebase instead of merge)?